apps/web/src/components/Shared/Markup/MarkupLink/ExternalLink.tsx

```diff
@@ -1,4 +1,4 @@
-import stopEventPropagation from '@hey/lib/stopEventPropagation';
+import { isUrlContainsValidTld } from '@hey/data/utils/check_valid_tld';
```

Comment:
move this to lib folder, data folder are all static files and values

Reply:
moved to lib